src/main/java/seedu/address/logic/commands/UnmarkAttendanceCommand.java

```diff
@@ -0,0 +1,123 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.MarkAttendanceCommand.findByTelegram;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TELEGRAM;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.Attendance;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Telegram;
+import seedu.address.model.role.Role;
+
+/**
+ * Unmark the attendance of people with targeted telegram handle.
+ */
+public class UnmarkAttendanceCommand extends AttendanceMarkingCommand {
+    public static final String COMMAND_WORD = "unmark";
+    public static final String COMMAND_ALIAS = "u";
```

Comment:
do you think 'um' will be a better alias? cus 'u' sounds like undo

Reply:
Thanks, changed to `um`